aliases/available/node.aliases.bash

```diff
@@ -0,0 +1,8 @@
+cite 'about-alias'
+about-alias 'the nodejs environment aliases'
+
+# alias to setup nodejs development environment
+alias node-dev='export NODE_ENV=development'
+
+# alias to setup nodejs production environment
+alias node-prod='export NODE_ENV=production'
```

Comment:
Please add a line break at the end of the file.

Reply:
Done